### ISSUE TITLE
"expo start" command doesn't work

### ISSUE THREAD

**Issue by yesIamFaded** [henninghall/react-native-date-picker#458]
I'have tried but I cannot reproduce it. 

Can you reproduce it when doing the following?

1. `expo init my-project`
2. `cd my-project`
3. `expo install react-native-date-picker expo-dev-client`
4. Add the code to display the DatePicker in `App.js`
5. `expo run:android`

_Originally posted by @henninghall in https://github.com/henninghall/react-native-date-picker/issues/454#issuecomment-1037302946_

I have build a new App with your given advice, so basically it has nothing just the DatePicker. I am still experiencing the same issues... So this is defo a Problem. 

Tried it with the newest Expo-CLI, SDK and the newest react-native-date-picker being version 4.1.5

Step by Step reproduction:
- basically only the steps you have already told me

**Comment by henninghall** [comment]
Thanks for helping med investigate this. 

1. Can publish a repo with this issue? 
2. What OS are you running on your computer?

**Comment by myhendry** [comment]
it seems we can no more run expo start after installing react-native-date-picker? we need to run on expo run:android to test our app and can no more use expo start?

**Comment by yesIamFaded** [comment]
I am not able to create a repo atm, but it is basically just a new expo project and then just install the module. Running the project with npm start or expo start. I am using Windows10.

**Comment by myhendry** [comment]
I don't think you can run expo start anymore. I tried using expo run: android after i install the module and it works ok. i posted a question on running expo start after the module installation. I think most likely cannot use expo start anymore post module installation

**Comment by yesIamFaded** [comment]
I can confirm that using the package with `expo start` does not work! Running it with `expo:run android` works.  The Problem is that, in my case, I need to test the app on Android and on IOS - so a functioning `expo start` would be needed for me.

**Comment by henninghall** [comment]
If `expo start` it doesn't work even after you have run `expo:run android` or `expo:run ios` I suspect that `expo start` was never intended to work when using custom clients. Anyway, this is qustion for Expo, i suggest you ask it there:
https://github.com/expo/expo
